src/sentry/api/endpoints/organization_events_stats.py

```diff
@@ -36,12 +36,16 @@ def get(self, request, organization):
 
         result = snuba.transform_aliases_and_query(
             skip_conditions=True,
+            aggregations=snuba_args.get("aggregations"),
```

Comment:
👍 Any risk we'll accidentally miss passing fields through here?

Reply:
Right now we don't have any use-cases that require non-aggregate fields to be used in a timeseries result.

Reply:
I mean more the switch from using kwargs to explicit params.

Reply:
I had to switch to explicit args because spreading `snuba_args` was causing duplicate keyword arguments. I thought this approach was simpler than popping out the potentially duplicate keys.

Reply:
👍  I actually prefer explicit to be clear